setup.py

```diff
@@ -251,9 +251,16 @@ def compile_KLU():
             "tqdm",
         ],
         "dev": [
-            "pre-commit",  # For code style checking
-            "ruff", # For code style auto-formatting
-            "nox", # For running testing sessions
+            # For working with pre-commit hooks
+            "pre-commit",
+             # For code style checks: linting and auto-formatting
+            "ruff",
+            # For running testing sessions
+            "nox",
+            # For testing Jupyter notebooks
+            "pytest",
```

Comment:
```suggestion
            "pytest>=6",
```

Reply:
Scientific Python guide says `>=6`

I'm not sure why, never looked at it in detail, but we can keep it `>=6`